src/api.v2/helpers/pipeline/gem2s.js

```diff
@@ -19,17 +19,35 @@ const logger = getLogger('[Gem2sService] - ');
 
 const hookRunner = new HookRunner();
 
-const saveProcessingConfigFromGem2s = ({ experimentId, item }) => {
-  logger.log('Saving processing config for gem2s');
-  new Experiment().updateById(experimentId, { processing_config: item });
-  logger.log('Finished saving processing config for gem2s');
-};
+const continueToQC = async (payload) => {
```

Comment:
shouldn't `continueToQC` actually be called `startQC`?

Reply:
I like `continueToQC` becuase it makes it clear that we have kind of concatenated qc after gem2s